tests/deb/deb_tests.bzl

```diff
@@ -37,16 +56,58 @@ def _package_naming_test_impl(ctx):
     if ctx.attr.expected_name:
         asserts.equals(
             env,
-            deb_path.split("/")[-1],  # basename(path)
             ctx.attr.expected_name,
+            deb_path.split("/")[-1],  # basename(path)
             "Deb package file name is not correct",
         )
     return analysistest.end(env)
 
-
 package_naming_test = analysistest.make(
     _package_naming_test_impl,
     attrs = {
         "expected_name": attr.string(),
     },
 )
+
+def _all_files_in_default_info_impl(ctx):
```

Comment:
Another way to implement this would be to convert everything to use skylib sets, and assert that that everything you want [is a subset of what's expected](https://github.com/bazelbuild/bazel-skylib/blob/1.0.3/docs/new_sets_doc.md#setsis_subset).

What you have here may also be easier to follow.  It's fine either way.

Reply:
It seems a waste to use is_subset for a singleton item.